articulationPoint.js

```diff
@@ -0,0 +1,123 @@
+const GraphBase = require('./graph-base');
+const Queue = require('./ds/queue');
+
+class ArticulationPoint extends GraphBase {
+    constructor(numVertices = 0, numEdges = 0, startingPoint = 0) {
+        super(numVertices, numEdges);
+    }
+
+    findArtiPt(graph, v, time) {
+        // console.log("hello from node "+v);
+        graph[v].visited = true;
+        time++;
+        graph[v].disc = time;
+        graph[v].low = time;
+        var child = 0;
+        // console.log(graph[v].al.length);
```

Comment:
Could you remove unused console.log statements?

Reply:
okay